src/main/scala/regmapper/Annotation.scala

```diff
@@ -21,3 +125,64 @@ object RegMappingAnnotation {
         ("regfields" -> regDescs)))))
   }
 }
+
+//case class RegFieldhHolder(named: Named, json: String)
+
+import java.io._
+
+@SerialVersionUID(123L)
+case class RegFieldDescSer(
+  name: String,
+  desc: String,
+  group: String,
+  groupDesc: String,
+  access: String,
+  wrType: String,
+  rdAction: String,
+  volatile: Boolean,
+  reset: BigInt
```

Comment:
not all registers have reset values. This is a pretty critical thing to capture if it's not reset.

Reply:
Should I add a hasRest boolean?

Reply:
Again, I don't know what you are doing and why the absence of a `resetValue` field in the emitted JSON is not allowed / sufficient. If there is no `resetValue`, that is the same as `hasReset = false`